Quaver.Shared/Screens/Gameplay/GameplayScreen.cs

```diff
@@ -523,6 +523,12 @@ private void HandlePauseInput(GameTime gameTime)
                 {
                     OffsetConfirmDialog.Exit(this);
                 }
+                // Exit the playing replay.
+                else if (InReplayMode)
+                {
+                    CustomAudioSampleCache.StopAll();
+                    Exit(() => new SelectScreen());
```

Comment:
Might want to be careful here and deactivate any mods that the play initially had on it. Example of this would be the "Paused" mod which is a hidden mod that dictates if a user paused throughout the play.